### DISCUSSION TITLE
SPARQL Star Parser

### DISCUSSION THREAD

**Discussion by GhinwaFAKIH** [apache/jena#1914] [Q&A] [answered]
Hello,
Where I can find the SPARQL-Star parser here? I can just find the SPARQL Parser only.
Thank you in advance

**Comment by afs** [comment, marked as answer]
There isn't a separate parser - the default SPARQL parser supports quoted triples as do the storage layers and the Turtle/N-triples/TriG and N-Quads parser/writers.

Jena supports the W3C CG final report: https://w3c.github.io/rdf-star/cg-spec

Details may change as the RDF-Star working group proceeds.
